src/index.html

```diff
@@ -17,6 +17,61 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link-logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#Apple"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a href="#Samsung">Samsung</a>
+          </li>
+
+          <li>
+            <a href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li>
+            <a
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Replace spaces with hyphens or underscores.